pubspec.yaml

```diff
@@ -3,23 +3,23 @@
 # BSD-style license that can be found in the LICENSE file.
 
 name: ffigen
-version: 2.0.0-dev.3
+version: 2.0.0-dev.4
 homepage: https://github.com/dart-lang/ffigen
 description: Experimental generator for FFI bindings, using LibClang to parse C header files.
 
 environment:
-  sdk: '>=2.12.0-198.0.dev <3.0.0'
+  sdk: ">=2.12.0-198.0.dev <3.0.0"
 
 dependencies:
   ffi: ^0.2.0-nullsafety.1
   yaml: ^3.0.0-nullsafety.0
   path: ^1.8.0-nullsafety.3
   quiver: ^3.0.0-nullsafety.2
-  args: ^1.6.0        # Not available
-  logging: ^0.11.4    # test->coverage->logging
-  cli_util: ^0.2.0    # test->analyzer->cli_util
-  glob: ^1.0.3        # test->analyzer->glob
-  pub_semver: ^1.4.4  # test->analyzer->pub_semver
+  args: ^1.6.0 # Not available
```

Comment:
Maybe undo these formatting changes.

If you want to do formatting changes, please make a separate PR and make the GitHub actions check the formatting.